code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -65,12 +65,21 @@
 	// when we get this many shards, we get a free bulb.
 	var/shards_required = 4
 
+//monkestation edit  (using brped)
+/obj/item/lightreplacer/afterattack(obj/item/itemtarget, mob/living/user)
```

Comment:
You have this double defined.
Make it work in the already existing afterattack proc.